.github/workflows/test.yml

```
@@ -25,9 +25,11 @@ jobs:
           flutter config --no-analytics
       - run: flutter pub get
       - run: flutter pub run webcrypto:setup
-      - run: flutter test
-      - run: flutter test --platform chrome
-      - run: xvfb-run flutter test integration_test/webcrypto_test.dart -d linux
+      - name: Run tests and collect coverage
+        run: |
+          flutter test --coverage
+          flutter test --platform chrome --coverage
+          xvfb-run flutter test integration_test/webcrypto_test.dart -d linux --coverage
```

I think we should keep them as different steps, so that it's easy to see which test failed. If a test actually fails.

It's also much easier if the display of the step we're running indicates what it is we're running.

That said, it's fine to exclude `--coverage` or `xvfb-run` from the display name.

Suggestion below:

```suggestion
      - name: flutter test
        run: flutter test --coverage
      - name: flutter test --platform chrome
        run: flutter test --platform chrome --coverage
      - name: flutter test integration_test/webcrypto_test.dart -d linux
        run: xvfb-run flutter test integration_test/webcrypto_test.dart -d linux --coverage
```

We should do similar for the other cases.